patches/server/0009-Adventure.patch

```diff
@@ -524,39 +523,59 @@ index 0000000000000000000000000000000000000000..8b7b140ac0326ffccd51684e28364714
 +    final class ViewersOutgoingChat implements OutgoingChat {
 +        @Override
 +        public void sendFormatChangedViewerAware(CraftPlayer player, Component displayName, Component message, ChatRenderer renderer, Set<Audience> viewers, ChatType.Bound chatType) {
-+            this.broadcastToViewers(viewers, player, chatType, v -> PaperAdventure.asVanilla(renderer.render(player, displayName, message, v)));
++            this.broadcastToViewers(viewers, chatType, v -> PaperAdventure.asVanilla(renderer.render(player, displayName, message, v)));
 +        }
 +
 +        @Override
 +        public void sendMessageChanged(CraftPlayer player, net.minecraft.network.chat.Component renderedMessage, Set<Audience> viewers, ChatType.Bound chatType) {
-+            this.broadcastToViewers(viewers, player, chatType, new ConstantFunction(renderedMessage));
++            this.broadcastToViewers(viewers, chatType, new ConstantFunction(renderedMessage));
```

Comment:
Do we need ConstantFunction now? I don't see it being used anymore. Can just be a regular `() -> renderedMessage` lambda.